addons/api/addon/generated/models/host-catalog.js

```diff
@@ -65,6 +65,13 @@ export default class GeneratedHostCatalogModel extends BaseModel {
 
   // AWS specific
 
+  @attr('string', {
+    for: 'plugin',
```

Comment:
What's the point of this `for` since we have to do a custom serializer to handle the `worker_filter` for azure plugins?